CHANGELOG.md

```diff
@@ -1,18 +1,39 @@
 # App Center SDK for Unity Change Log
 
-## Version 4.0.1 (under development)
+## Release 4.1.0
 
-### App Center Distribute
+### AppCenter
+
+* **[Feature]** Add a `SetMaxStorageSizeAsync` API which allows setting a maximum size limit on the local SQLite storage. The default value is 10MiB.
```

Comment:
UWP only.
And it is not work at the moment - https://github.com/microsoft/appcenter-sdk-unity/blob/develop/Assets/AppCenter/Plugins/AppCenterSDK/Core/UWP/AppCenterInternal.cs#L195

Need to fix it before continue the release

Reply:
Thanks for the reminder. I've solved it